//                     A little bug patched in jail 1.9 (it didn't set the shell correctly in /etc/shells)
//                                                     v1.0a
//                                                   by Bugghy (bugghy@rootshell.be)

*** ../lib/arch/generic/functions      Fri Oct 26 12:36:09 2001
--- ../lib/arch/generic/functions      Sat Jun 14 13:41:22 2003
***************
*** 823,829 ****
    close(F);
  
    for $k (@elem) {
!     if ($k =~ /^$INSTALL_DIR\/jail/) {
        $found = 1;
        last;
      }
--- 823,829 ----
    close(F);
  
    for $k (@elem) {
!     if ($k =~ /^$INSTALL_DIR\/bin\/jail/) {
        $found = 1;
        last;
      }
***************
*** 831,837 ****
  
    if (!$found || !-e $SHELLS_FILE) {
      open(F,">>".$SHELLS_FILE);
!     print F "$INSTALL_DIR/jail\n";
      close(F);
    } 
  }
--- 831,837 ----
  
    if (!$found || !-e $SHELLS_FILE) {
      open(F,">>".$SHELLS_FILE);
!     print F "$INSTALL_DIR/bin/jail\n";
      close(F);
    } 
  }
